Lib/fontbakery/profiles/fontwerk.py

```diff
@@ -31,22 +36,39 @@ def leave_this_one_out(checkid):
         'com.google.fonts/check/vendor_id',
         'com.google.fonts/check/fstype',
         'com.google.fonts/check/gasp',
+        'com.google.fonts/check/font_copyright',
+        'com.google.fonts/check/name/line_breaks',  # can be ignored because of general fontwerk name table test.
+        'com.google.fonts/check/varfont/has_HVAR',  # not needed for fontwerk fonts.
+        'com.google.fonts/check/glyf_nested_components',  # it seem to be not an issue anymore. Maybe downgrade to 'WARN'. Tested it in various browsers, InDesign, exported a PDF, printed it.
+        'com.google.fonts/check/transformed_components',  # it seem to be not an issue anymore. Maybe downgrade to 'WARN'.
```

Comment:
This issue is interesting in that regard: https://github.com/googlefonts/gftools/issues/424#issuecomment-942092920